src/distributions/range.rs

```diff
@@ -314,6 +314,19 @@ macro_rules! range_int_impl {
                 }
             }
         }
+
+        impl From<::core::ops::Range<$ty>> for RangeInt<$ty> {
+            fn from(r: ::core::ops::Range<$ty>) -> RangeInt<$ty> {
+                RangeInt::<$ty>::new(r.start, r.end)
+            }
+        }
+
+        impl From<::core::ops::Range<$ty>> for Range<$ty>
```

Comment:
Maybe make this a generic implementation for all `RangImpl`s?
```rust
impl<X> From<::core::ops::Range<X>> for Range<X>
    where X: SampleRange,
          <X as SampleRange>::Impl: From<::core::ops::Range<X>>
{
    fn from(r: ::core::ops::Range<X>) -> Range<X> {
        Range { inner: <X as SampleRange>::Impl::from(r) }
    }
}
```

Reply:
Thinking about it a bit more, a simple generic implementation is all that is needed, no implementation on `RangeImpl` is necessary:
```rust
impl<X: SampleRange> From<::core::ops::Range<X>> for Range<X> {
    fn from(r: ::core::ops::Range<X>) -> Range<X> {
        Range::new(r.start, r.end)
    }
}
```

Reply:
Sure!